packages/studio-base/src/panels/ThreeDeeRender/stories/foxglove.Grid.stories.tsx

```diff
@@ -0,0 +1,522 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import { Grid, NumericType } from "@foxglove/schemas";
+import { MessageEvent, Topic } from "@foxglove/studio";
+import { LayerSettingsFoxgloveGrid } from "@foxglove/studio-base/panels/ThreeDeeRender/renderables/FoxgloveGrid";
+import PanelSetup from "@foxglove/studio-base/stories/PanelSetup";
+
+import ThreeDeeRender from "../index";
+import { TransformStamped } from "../ros";
+import { QUAT_IDENTITY, rad2deg } from "./common";
+import useDelayedFixture from "./useDelayedFixture";
+
+export default {
+  title: "panels/ThreeDeeRender",
+  component: ThreeDeeRender,
```

Comment:
```suggestion
  component: ThreeDeeRender,
  parameters: { colorScheme: "light" },
```
removes the double-rendering of each story with two color schemes because the color scheme isn't really relevant to most 3d panel stories.